examples/mda_widget.py

```diff
@@ -7,78 +7,64 @@
     QVBoxLayout,
     QWidget,
 )
-from useq import MDAEvent, MDASequence
+from useq import MDAEvent
 
 from pymmcore_widgets import MDAWidget
 
 
 class MDA(QWidget):
     """An example of using the MDAWidget to create and acquire a useq.MDASequence.
-
     The `MDAWidget` provides a GUI to construct a `useq.MDASequence` object.
     This object describes a full multi-dimensional acquisition;
-
     In this example, we set the `MDAWidget` parameter `include_run_button` to `True`,
     meaning that a `run` button is added to the GUI. When pressed, a `useq.MDASequence`
     is first built depending on the GUI values and is then passed to the
     `CMMCorePlus.run_mda` to actually execute the acquisition.
-
     For details of the corresponding schema and methods, see
     https://github.com/pymmcore-plus/useq-schema and
     https://github.com/pymmcore-plus/pymmcore-plus.
-
     In this example, we've also connected callbacks to the CMMCorePlus object's `mda`
     events to print out the current state of the acquisition.
-    """
+    """  # noqa: D205
```

Comment:
why did you get rid of these line breaks and add a noqa?

Reply:
oh maybe that was just how it was in widget2?

Reply:
fixed

Reply:
fixed